src/android/io_realm_react_RealmReactModule.cpp

```diff
@@ -93,7 +93,7 @@ JNIEXPORT void JNICALL Java_io_realm_react_RealmReactModule_setDefaultRealmFileD
 
     // Setting the internal storage path for the application
     const char* strFileDir = env->GetStringUTFChars(fileDir, NULL);
-    realm::set_default_realm_file_directory(strFileDir);
+    realm::JsPlatformHelpers::set_default_realm_file_directory(strFileDir);
```

Comment:
🤔 (how) did this work before?

Reply:
Never mind - I understand think I understand it better now.

Reply:
Nope, but since we do late binding, you will only see if it use the method